### DISCUSSION TITLE
Storage metadata availability in GraphQl

### DISCUSSION THREAD

**Discussion by DanishSiraj** [nhost/nhost#21] [Ideas]
I was wondering if it would be possible to add the name and url of storage items into the database by default so that they become easily available in graphql.

**Comment by elitan** [comment]
It's not available right now but I like this suggestion. This would require lots of internal changes for our Nhost Storage which we're discussing internally right now.

**Reply by elitan** [reply]
Open discussion about this here: https://github.com/nhost/hasura-backend-plus/discussions/516
